projects/components/src/datagrid/filters/datagrid-string-filter.component.ts

```diff
@@ -35,6 +35,22 @@ export class DatagridStringFilterComponent extends DatagridFilter<string, Datagr
         });
     }
 
+    /**
+     * Creates a {@link FilterRendererSpec} with the given config.
+     */
+    static factory(
```

Comment:
Please rename to `create` and specify `@param` in the docs

Reply:
Added documentation for params. But the reason I didn't name it `create` is because any class that extends SubscriptionTrackerMixin(Object) inherits all of Objects static methods. Object comes with a `create` method that clashes when I try and add this `create` method. Totally open to other names however.

Reply:
Let's export a separate function that is named by removing the `Component` suffix

`DatagridStringFilter(params)`